Python/optimizer_symbols.c

```diff
@@ -0,0 +1,332 @@
+
+#include "Python.h"
+
+#include "cpython/optimizer.h"
+#include "pycore_code.h"
+#include "pycore_frame.h"
+#include "pycore_optimizer.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stddef.h>
+
+// Flags for below.
```

Comment:
The comment is out of date -- the struct containing the flags has moved to pycore_optimizer.h, but the flag bits are still here (a useful abstraction to prevent circumventing the official API).